install.zasilkovna.php

```diff
@@ -104,6 +104,7 @@ public function preflight($route, $adapter) {
             recurse_delete($media_path);
 
             $this->removeAdministratorFiles();
+            $this->removeAlzaboxFromShipmentParams();
```

Comment:
Máme možnost spouštět to jen při upgrade na tuto verzi, nebo předpokládáme, že to tam bude navždy?

Reply:
Pokud vim, verze se splitem nebyla nikdy vydana, takze toto resi ty okrajove pripady, ze si nekdo nainstaloval master a dopravni metody si se splitem nakonfiguroval.

A protoze updatovat plugin muzes i preskocenim nekolika verzi, tak to tam bude tak dlouho, dokud si nerekneme, ze uz je to dost dlouho a odstranime to.

Reply:
Bude řešeno v PES-2028.